api/urls.py

```diff
@@ -75,13 +75,21 @@ def to_url(self, value):
     api_router.register(r"relaynumber", RelayNumberViewSet, "relay_number")
     api_router.register(r"inboundcontact", InboundContactViewSet, "inbound_contact")
     urlpatterns += [
+        # TODO: Update Twilio webhooks to versions with trailing slashes,
+        #       then remove versions without trailing slashes (Django's
+        #       APPEND_SLASH option will then make those redirect).
         path("v1/inbound_sms", inbound_sms, name="inbound_sms"),
+        path("v1/inbound_sms/", inbound_sms, name="inbound_sms"),
```

Comment:
Nit: Remove duplicate names

I was surprised that you can re-use the name `"inbound_sms"`, but I believe this is how you can override a URL handler from a third party library, so it makes sense that names can be reused, in which case the last one wins.